README.md

```diff
@@ -1,12 +1,46 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
-<!-- Copyright Contributors to the Egeria project. -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
 
-# Egeria
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3044/badge)](https://bestpractices.coreinfrastructure.org/projects/3044)
 
-<!-- TODO: Add a README -->
+[//]: # ([![Azure]&#40;https://dev.azure.com/odpi/egeria/_apis/build/status/odpi.egeria&#41;]&#40;https://dev.azure.com/odpi/Egeria/_build&#41;)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=odpi_egeria&metric=alert_status)](https://sonarcloud.io/dashboard?id=odpi_egeria)
```

Comment:
Remove - we're not using sonarcloud in this project

Also Maven central points to wrong artifact. I would recommend to change to the name of what we are building here if useful, else remove